src/app/modules/user/settings/settings.component.html

```diff
@@ -0,0 +1 @@
+<p>settings works!</p>
```

Comment:
let's not have this because main branch will be the deployed branch where everything is as it should be, not for testing purposes. We def not need to have this in the main branch